src/app/draft/page.tsx

```
@@ -1,603 +1,91 @@
+import { RecentMarkets } from './components/recent'
 import { MainLayout } from '@/components'
-import { useToast } from '@/hooks'
-import { useCategories, useLimitlessApi } from '@/services'
-import { useAxiosPrivateClient } from '@/services/AxiosPrivateClient'
-import { Category } from '@/types'
-import { Token, Tag, TagOption, IFormData, Creator } from '@/types/draft'
+import CopyIcon from '@/resources/icons/copy-icon.svg'
+import LoadingIcon from '@/resources/icons/loader-icon.svg'
+import PlusIcon from '@/resources/icons/plus-square-icon.svg'
+
+type Tab = {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Tab'. Consider to delete it or rename it.

'Tab' is defined here:


(lint/suspicious/noRedeclare)